memory_leak_worker_logging/evm_mem_log_processor.rb

```diff
@@ -43,13 +43,19 @@
   /\.([0-9]*) /,                       # 4. .ms
   /#([0-9]*)/,                         # 5. #pid
   /:([a-z0-9]*)\] +/,                  # 6. :thread
-  /[A-Z]{0,5} -- [^:]*: /,
+  /[A-Z]{0,5} -- [^:]*: /,             #    severity
```

Comment:
Curious if we think we should make this a match here... I never use it myself, but I think most of the comments here were designed to comment the things we were capturing, so curious if you think there would ever be a need for having the `severity` instead of the `severity char`?

I have no problem with this comment, for what it is worth, it is just that this one is now an odd one out because it is just informational, instead of documenting the Regexp matches.

Reply:
I think we should remove severity char